File: app/src/main/java/com/article/kotlinsynthetics/b_java_butterknife/MainActivity.java
package com.article.kotlinsynthetics.b_java_butterknife;

import android.os.Bundle;

import com.article.kotlinsynthetics.R;
import com.article.kotlinsynthetics.d_kotlin_synthetics.MainFragment;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main_activity);

        getSupportActionBar().setTitle("Java - ButterKnife");

        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.container, MainFragment.Companion.newInstance())
                    .commitNow();
        }
    }
}

File: app/src/main/java/com/article/kotlinsynthetics/c_kotlin_butterknife/MainActivity.kt
package com.article.kotlinsynthetics.c_kotlin_butterknife

import android.os.Bundle

import com.article.kotlinsynthetics.R

import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main_activity)

        supportActionBar!!.title = "Kotlin - ButterKnife"

        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction().replace(R.id.container, MainFragment.newInstance()).commitNow()
        }
    }
}
